mods/ra/rules/vehicles.yaml

```diff
@@ -733,6 +736,9 @@ QTNK:
 		RevealGeneratedShroud: False
 	RevealsShroud@GAPGEN:
 		Range: 4c0
+	RevealOnDeath:
```

Comment:
I'm not a fan of having this universally enabled - if someone kills a QTNK when it's driving around it should behave the same way as any other killed unit.  A simple fix would be to enable RevealOnDeath for everything.